src/containers/Admin/LogoTab/CustomURL/index.tsx

```diff
@@ -0,0 +1,142 @@
+import React, { useState } from 'react'
+
+import {
+    Label,
+    Heading3,
+    Paragraph,
+    EmphasizedText,
+    StrongText,
+} from '@entur/typography'
+import { TextField } from '@entur/form'
+import { SecondarySquareButton } from '@entur/button'
+import { CheckIcon } from '@entur/icons'
+import { Tooltip } from '@entur/tooltip'
+
+import {
+    copySettingsToNewId,
+    setIdToBeDeleted,
+} from '../../../../services/firebase'
+
+import { useSettingsContext } from '../../../../settings'
+import { getDocumentId } from '../../../../utils'
+
+import '../styles.scss'
+
+enum inputFeedback {
+    ID_UNAVAILABE = 'Ikke ledig: Denne Tavla-linken er dessverre i bruk.',
+    ID_SET = 'Din nye Tavla-link er nå opprettet!',
+    EMPTY_STRING = 'Tomt felt: Tavle-linken kan ikke være tom.',
+    NOTHING = '',
+}
+
+enum inputFeedbackClass {
+    SUCCSESS = 'succsessMessage',
+    FAILURE = 'errorMessage',
+}
+
+const CustomURL = (): JSX.Element => {
+    const settings = useSettingsContext()
+
+    const [customUrlInput, setCustomUrlInput] = useState('')
+    const [feedbackMessage, setFeedbackMessage] = useState(
+        inputFeedback.NOTHING,
+    )
+    const [feedbackMessageClass, setFeedbackMeessageClass] = useState(
+        inputFeedbackClass.FAILURE,
+    )
+    const [currentDoc, setCurrentDoc] = useState(getDocumentId() as string)
+
+    const handleCustomUrlChange = (
+        event: React.ChangeEvent<HTMLInputElement>,
+    ): void => {
+        if (event.currentTarget.value.match(/[^A-Za-z0-9_-]/g)) {
+            event.currentTarget.value = event.currentTarget.value.replace(
+                /[^A-Za-z0-9_-]/g,
+                '',
+            )
+        }
+        setCustomUrlInput(event.target.value)
+        if (feedbackMessage) {
+            setFeedbackMessage(inputFeedback.NOTHING)
+        }
+    }
+
+    const tryAddCustomUrl = () => {
+        if (!customUrlInput) {
```

Comment:
Burde me også ha reglar på f.eks. minimumslengde? Ser for meg at dei kortaste URLane kan bli populære. Kanskje det er fint å kreve >6 tegn sånn at kortare URLer berre kan utdelast av oss? Då unngår me "domenekupping" og har litt meir kontroll.

Reply:
Høres lurt ut! Satte sånn at lenken må være på minst seks tegn nå